source/element-kinds.md

```diff
@@ -412,10 +417,14 @@ Element parameter groups associated with this element kind are:
 - [**BodyShiftP**](#s:bodyshift.params): Orientation of element with respect to its nominal position.
 - [**FloorP**](#s:floor.params): Floor position and orientation.
 - [**MetaP**](#s:meta.params): Meta parameters.
+- [**PatchP**](#s:meta.params): Exit coordinates with respect to entrance coordinates.
 - [**ReferenceP**](#s:ref.params): Reference parameters.
 - [**ReferenceChangeP**](#s:refchange.params): Reference energy change and/or reference time correction.
 - [**TrackingP**](#s:tracking.params): Tracking parameters.
 
+Important: By convention, any energy shift is applied after a particle reaches the exit face.
+This matters when the speed of the reference particle is less than the speed of light.
```

Comment:
Since this is always the case, maybe this sentence should be rephrased by changing "less" to "significantly less", or something more quantitative.